roles/edpm_telemetry/tasks/install.yml

```diff
@@ -131,36 +131,6 @@
       [Install]
       WantedBy=multi-user.target
```

Comment:
as @SeanMooney mentioned, there is a role for managing systemd units for containers, could you please use it instead?